node/network/availability-distribution/src/tests.rs

```diff
@@ -22,18 +22,25 @@ use polkadot_node_subsystem_util::TimeoutExt;
 use polkadot_primitives::v1::{
 	AvailableData, BlockData, CandidateCommitments, CandidateDescriptor, GroupIndex,
 	GroupRotationInfo, HeadData, OccupiedCore, PersistedValidationData, PoV, ScheduledCore,
+	ValidatorId,
 };
 use polkadot_subsystem_testhelpers as test_helpers;
 
 use futures::{executor, future, Future};
 use sc_keystore::LocalKeystore;
 use sp_application_crypto::AppKey;
 use sp_keystore::{SyncCryptoStore, SyncCryptoStorePtr};
+use sp_keyring::Sr25519Keyring;
 use std::{sync::Arc, time::Duration};
-use maplit::hashmap;
+use maplit::{hashset, hashmap};
 
 macro_rules! view {
-	( $( $hash:expr ),* $(,)? ) => { View(vec![ $( $hash.clone() ),* ]) };
+	( $( $hash:expr ),* $(,)? ) => {
+		View(vec![ $( $hash.clone() ),* ])
+	};
+	[ $( $hash:expr ),* $(,)? ] => {
+		View(vec![ $( $hash.clone() ),* ])
+	};
```

Comment:
These both branches are doing the same?

Reply:
one for `[]` one for `()`, but other than that, yes.

Reply:
This is not required. Please remove the second branch.